gwdetchar/lasso/core.py

```diff
@@ -33,61 +33,91 @@
 
 # -- utilities ----------------------------------------------------------------
 
-def find_outliers(ts, N=5):
+def find_outliers(ts, L=5, method='s'):
```

Comment:
I suggest we keep the `N` kwarg nomenclature; downstream users will see their code break otherwise